pymarlin/core/trainer.py

```diff
@@ -140,11 +143,38 @@ def __init__(
         self.trainer_backend.init(self._get_trainer_backend_args())
         self.trainer_backend.update_state(self.checkpointed_states.trainer_backend_state)
 
+        if self.args.ort: 
+            from torch_ort import ORTModule 
```

Comment:
This is not good design. Why do you need to wrap `module` in trainer instead of creating a new `trainer_backend`?